app/src/main/java/com/hedvig/app/feature/marketing/pickmarket/PickMarketScreen.kt

```diff
@@ -176,14 +180,14 @@ fun BottomSheetHandle(modifier: Modifier = Modifier) {
         color = MaterialTheme.colors.separator,
         shape = RoundedCornerShape(20.dp),
       ),
-
   )
 }
 
 @Composable
 fun PickMarketSheetContent(
   onSelectMarket: (Market) -> Unit,
-  data: PickMarket,
+  selectedMarket: Market?,
```

Comment:
I like this thing you're doing with composables only receiving the thing they need instead of a bigger object which may have things the composable doesn't need. I think in general it is a good idea, expect for the "top level" if that makes sense composables which do need the whole object.